app/models/katello/docker_tag.rb

```diff
@@ -30,6 +38,16 @@ def update_from_json(json)
       self.save!
     end
 
+    def self.import_all(uuids = nil, options = {})
```

Comment:
Does this need to get updated? https://github.com/Katello/katello/blob/master/app/models/katello/glue/pulp/repo.rb#L757-L759